src/sas/qtgui/Utilities/TabbedModelEditor.py

```diff
@@ -455,45 +512,42 @@ def updateFromEditor(self):
 
         # Get model filepath
         plugin_location = models.find_plugins_dir()
-        full_path = os.path.join(plugin_location, filename)
-        if not w.is_python and self.is_python:
-            pass
-        elif os.path.splitext(full_path)[1] != ".py":
-            full_path += ".py"
-
-        # Check model as long as there is a .py file in one of the tabs
-        if w.is_python and self.is_python:
-            check_model = True
-        elif not w.is_python and self.is_python:
-            # Set full_path to the .py file so that we can run a model check on it (the .py model should link to the .c model)
-            full_path = self.filename.with_suffix(".py")
-            check_model = True
-        
-        if check_model:
-            error_line = self.findFirstError(full_path)
-            if error_line >= 0:
-                # select bad line
-                cursor = QtGui.QTextCursor(w.txtEditor.document().findBlockByLineNumber(error_line-1))
-                w.txtEditor.setTextCursor(cursor)
-                return
+        full_path = Path(plugin_location) / filename
 
-        # change the frame colours back
-        try:
-            self.c_editor_widget.txtEditor.setStyleSheet("")
-            self.c_editor_widget.txtEditor.setToolTip("")
-        except AttributeError:
-            pass
-        self.editor_widget.txtEditor.setStyleSheet("")
-        self.editor_widget.txtEditor.setToolTip("")
+        if self.is_python and self.findFirstError(full_path.with_suffix(".py")) >= 0:
+            # select bad line
+            cursor = QtGui.QTextCursor(w.txtEditor.document().findBlockByLineNumber(error_line-1))
```

Comment:
Here, `error_line` is not defined. I'm getting errors related to this when testing.